pandas/core/series.py

```diff
@@ -3234,40 +3239,25 @@ def _try_cast(arr, take_fast_path):
     else:
         subarr = _try_cast(data, False)
 
-    # scalar like, GH
-    if getattr(subarr, 'ndim', 0) == 0:
-        if isinstance(data, list):  # pragma: no cover
-            subarr = np.array(data, dtype=object)
-        elif index is not None:
-            value = data
+    if subarr.ndim == 0 or is_scalar(data):
+        if index is None:
+            return subarr.item()
```

Comment:
this is very odd, are you actually returning a scalar on construction?

Reply:
I have no idea of what could possibly hit this (i.g. when ``index is None``), and it does look broken (e.g. ``BlockManager`` won't accept it), so I removed it, but if you want to stay on the safe side just discard the last commit I just pushed.